modules/ROOT/pages/feature-flagging.adoc

```diff
@@ -747,6 +751,22 @@ This feature isn't configurable by system property.
 *Issue ID*
 
 * W-15750334
+<.^|`mule.ntlm.avoid.send.payload.on.type1`
+|When enabled, body contents aren't sent on NTLM type 1 requests. This saves resources by not sending a payload that is never consumed (the server rejects the payload until the dance is completed).
```

Comment:
Is there a typo in "dance"?

Reply:
"dance" is used to describe the orchestration of steps.